strategy-service.ts

```diff
@@ -0,0 +1,5 @@
+import { strategyType, type Candle } from "./types/common";
+
+function strategyService(candle: Candle): strategyType {
```

Comment:
Function names should be imperative most of the times, e.g. `evaluateAllStrategies` or something.